code/modules/fishing/fishing_gear.dm

```diff
@@ -350,3 +350,341 @@ TYPEINFO(/obj/item/fish_portal)
+			if (src.lure.owner && isliving(src.lure.owner))
+				logTheThing(LOG_COMBAT, user, "at [log_loc(src)] reels in a Syndicate Fishing Rod hooked in [src.lure.owner]")
+				if (!actions.hasAction(user,"fishing_for_fools"))
+					actions.start(new /datum/action/bar/syndie_fishing(user, src.lure.owner, src, src.lure), user)
+				if (!ON_COOLDOWN(user, "syndie_fishing_yank", src.yank_cooldown))
+					src.lure.owner.throw_at(target, yank_range, yank_range / 4)
+					user.visible_message("<span class='alert'><b>[user] thrashes [src.lure.owner] by yanking \the [src.name]!</b></span>")
+			else if (src.lure.loc == src)
+				if (target == loc)
+					return
+				logTheThing(LOG_COMBAT, user, "casts a Syndicate Fishing Rod out at [log_loc(src)]")
+				playsound(user, 'sound/items/fishing_rod_cast.ogg', 50, 1)
+				src.is_fishing = TRUE
+				src.UpdateIcon()
+				user.update_inhands()
+				src.lure.pixel_x = rand(-12, 12)
+				src.lure.pixel_y = rand(-12, 12)
+				src.lure.set_loc(get_turf(src.loc))
+				src.lure.throw_at(target, src.line_length, 2)
+			else
+				src.is_fishing = FALSE
+				src.UpdateIcon()
+				user.update_inhands()
+				src.lure.throw_at(src, 15, 2)
+				SPAWN(0.2 SECONDS)
+					if (src.lure)
+						if (src.lure.owner)
+							src.lure.owner.throw_at(src, 2, 2)
+						else
+							src.lure.set_loc(src)
+
+	update_icon()
+		//state for fishing
+		if (src.is_fishing)
+			src.icon_state = "syndie_fishing_rod-active"
+			src.item_state = "syndie_fishing_rod-active"
+		//state for not fishing
+		else
+			src.icon_state = "syndie_fishing_rod-inactive"
+			src.item_state = "syndie_fishing_rod-inactive"
+
+	disposing()
+		. = ..()
+		qdel(src.lure)
+
+	// reels in, returns whether damage was dealt
+	proc/reel_in(mob/target, mob/user, damage_on_reel = 7)
+		target.setStatusMin("staggered", 4 SECONDS)
+		if(BOUNDS_DIST(target, user) == 0)
+			if (issilicon(target))
+				user.visible_message("<span class='alert'><b>[user] tears some scrap out of [target] with \the [src.name]!</b></span>")
+				playsound(target.loc, 'sound/impact_sounds/circsaw.ogg', 40, 1)
+				random_burn_damage(target, damage_on_reel)
+			else
+				user.visible_message("<span class='alert'><b>[user] reels some meat out of [target] with \the [src.name]!</b></span>")
+				playsound(target.loc, 'sound/impact_sounds/Flesh_Tear_2.ogg', 50, 1)
+				take_bleeding_damage(target, user, damage_on_reel, DAMAGE_CUT)
+			random_brute_damage(target, damage_on_reel)
+			if (damage_on_reel >= 25)
+				target.changeStatus("weakened", sqrt(damage_on_reel) / 3 SECONDS)
+				target.force_laydown_standup()
+				if (target.bioHolder && target.bioHolder.Uid && target.bioHolder.bloodType)
+					gibs(target.loc, blood_DNA=target.bioHolder.Uid, blood_type=target.bioHolder.bloodType, headbits=FALSE, source=target)
+				else
+					gibs(target.loc, headbits=FALSE, source=target)
+			return TRUE
+		else
+			step_towards(target, user)
+			return FALSE
+
+/obj/item/implant/syndie_lure
```

Comment:
So. What happens when you load this into an implant gun.

Reply:
Oh, of course.  Thank you, I'll fix that!

Reply:
I've changed the lure to a generic item and stopped piggybacking on implant code.  If it's desired that a knife or something can remove the lure, I'll find a way to do that!